### PR TITLE
feat: translation storage services

### PR DESCRIPTION
**Pull request by MdechampG**
## Translation storage services

- [x] I've read the last version of [CODE_OF_CONDUCT.md](https://github.com/okp4/.github/blob/main/CODE-OF-CONDUCT.md) and [CONTRIBUTING.md](https://github.com/okp4/.github/blob/main/CONTRIBUTING.md)

### Description

I have added the translation in French and German for the storage services titles so that they can appear in the Portal.

### CODE REVIEW

File: example/rhizome/service/03230ce7-d8cb-410e-919e-19c480c1dd75.ttl [hunk 1]
```diff
@@ -14,6 +14,8 @@
     core:hasPublisher "OKP4" ;
     core:hasTag "stockage" ;
     core:hasTitle "Clever Cloud Data Storage"@en .
+    core:hasTitle "Clever Cloud Data Storage"@fr .
```
Comment (ccamel) [inline example/rhizome/service/03230ce7-d8cb-410e-919e-19c480c1dd75.ttl:17]: Syntax is not correct. Ending lines with `;` is required, except for the final line, which should end with `.`.

(this was detected by the CI but the error is not correctly reported)

**Review by ccamel — CHANGES_REQUESTED** [review]

File: example/rhizome/service/03230ce7-d8cb-410e-919e-19c480c1dd75.ttl (hunk 1 above)
Comment (MdechampG) [reply example/rhizome/service/03230ce7-d8cb-410e-919e-19c480c1dd75.ttl]: Thx I did the change.

**Review by ccamel — APPROVED** [review]
All good! Thanks :+1:

**Review by lolottetheclash — APPROVED** [review]
great many thanks !

**Pull request merged**